File: player.py
class Player:
    def __init__(self, name):
        self.name = name
        self.health = 100
        self.experience = 0
        self.inventory = []

    def take_damage(self, amount):
        self.health -= amount
        print(f"{self.name} получил {amount} урона! Здоровье: {self.health}")

    def gain_experience(self, amount):
        self.experience += amount
        print(f"{self.name} получил {amount} опыта! Опыт: {self.experience}")

    def add_to_inventory(self, item):
        self.inventory.append(item)
        print(f"{self.name} получил {item.name} в инвентарь!")

File: player.java
import java.util.ArrayList;
import java.util.List;

public class Player {
    private String name;
    private int health;
    private int experience;
    private List<Item> inventory;

    public Player(String name) {
        this.name = name;
        this.health = 100;
        this.experience = 0;
        this.inventory = new ArrayList<>();
    }

    public String getName() {
        return name;
    }

    public int getHealth() {
        return health;
    }

    public int getExperience() {
        return experience;
    }

    public List<Item> getInventory() {
        return inventory;
    }

    public void takeDamage(int amount) {
        health -= amount;
        System.out.println(name + " получил " + amount + " урона! Здоровье: " + health);
    }

    public void gainExperience(int amount) {
        experience += amount;
        System.out.println(name + " получил " + amount + " опыта! Опыт: " + experience);
    }

    public void addToInventory(Item item) {
        inventory.add(item);
        System.out.println(name + " получил " + item.getName() + " в инвентарь!");
    }
}
